app/views/cost_items/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="form-group <%= 'form-group--error' if f.object.errors[:name].any? %>">
```

Comment:
Maybe, my worry would be that it would get complex fast and the reduction in duplication would not justify the increase in complexity. 

We do have this pattern in a few places so it may be worth investigating if we can build something, but I would want that to be done separately.